bin/interactive.py

```diff
@@ -150,7 +150,7 @@ def get_validator_command():
             validator_err.decode('utf-8', 'replace'),
             exec_res.err,
             verdict,
-            last_pass,
+            last_pass if run.problem.multipass else None,
```

Comment:
It feels cleaner to always set this and instead do this `if` check where it's used/displayed.

Reply:
I think its cleaner to only set `pass_id` if we are doing a pass (and not judging a normal problem)